proxy/pkg/command/server.go

```diff
@@ -247,73 +246,67 @@ func Server(cfg *config.Config) *cli.Command {
 }
 
 func loadMiddlewares(ctx context.Context, l log.Logger, cfg *config.Config) alice.Chain {
-
-	psMW := middleware.PresignedURL(
-		middleware.Logger(l),
-		middleware.Store(storepb.NewStoreService("com.owncloud.api.store", grpc.NewClient())),
-		middleware.PreSignedURLConfig(cfg.PreSignedURL),
-	)
-
-	accounts := acc.NewAccountsService("com.owncloud.api.accounts", ogrpc.DefaultClient)
-	roles := settings.NewRoleService("com.owncloud.api.settings", ogrpc.DefaultClient)
-
-	uuidMW := middleware.AccountUUID(
-		middleware.Logger(l),
-		middleware.TokenManagerConfig(cfg.TokenManager),
-		middleware.AccountsClient(accounts),
-		middleware.SettingsRoleService(roles),
-		middleware.AutoprovisionAccounts(cfg.AutoprovisionAccounts),
-		middleware.EnableBasicAuth(cfg.EnableBasicAuth),
-		middleware.OIDCIss(cfg.OIDC.Issuer),
-	)
-
-	// the connection will be established in a non blocking fashion
-	sc, err := cs3.GetGatewayServiceClient(cfg.Reva.Address)
+	accountsClient := acc.NewAccountsService("com.owncloud.api.accounts", mclient.DefaultClient)
```

Comment:
Afaict all occurrences of `mclient.DefaultClient` should now be replaced with `grpc.DefaultClient`, with the grpc package coming from `"github.com/owncloud/ocis/ocis-pkg/service/grpc"`, as of https://github.com/owncloud/ocis/pull/840